src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Tokens/SessionSecurityToken.cs

```diff
@@ -915,7 +915,9 @@ private ClaimsIdentity ReadIdentity(XmlDictionaryReader dictionaryReader, Sessio
                 byte[] bytes = dictionaryReader.ReadContentAsBase64();
                 using (MemoryStream ms = new MemoryStream(bytes))
                 {
+#pragma warning disable SYSLIB0011
```

Comment:
Compiler is throwing error about BinaryFormatter being deprecated, this silences it